landlab/components/depth_dependent_cubic_soil_creep/hillslope_depth_dependent_cubic_flux.py

```diff
@@ -181,32 +261,84 @@ def __init__(self,grid,
         self._active_nodes = self.grid.status_at_node != CLOSED_BOUNDARY
 
 
-    def soilflux(self, dt):
+    def soilflux(self, dt, dynamic_dt=False, if_unstable='pass', courant_factor=0.2):
         """Calculate soil flux for a time period 'dt'.
```

Comment:
@mcflugen I used if_unstable = 'pass' (or 'raise', or 'warn') 
since using 
```python
pass
raise
warn
```
didn't seem possible. 

Is this what was intended by your recommendation?